torch/quantization/observer.py

```diff
@@ -649,10 +651,26 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                     self.min_vals.resize_(val.shape)
                 else:
                     self.max_vals.resize_(val.shape)
+                # For torchscript module we need to update the attributes here since we do not
+                # call the `_load_from_state_dict` function defined module.py
+                if torch.jit.is_scripting():
+                    if name == 'min_vals':
+                        self.min_vals = val
+                    else:
+                        self.max_vals = val
             elif strict:
                 missing_keys.append(key)
-        super(PerChannelMinMaxObserver, self)._load_from_state_dict(state_dict, prefix, local_metadata, strict,
-                                                                    missing_keys, unexpected_keys, error_msgs)
+
+        if not torch.jit.is_scripting():
+            super(PerChannelMinMaxObserver, self)._load_from_state_dict(state_dict, prefix, local_metadata, strict,
+                                                                        missing_keys, unexpected_keys, error_msgs)
+
+    @torch.jit.export
+    def _load_from_state_dict_script(self, state_dict: Union[Dict[str, torch.Tensor], Dict[str, torch.Tensor]],
```

Comment:
this seems to call `_load_from_state_dict`, just curious on what would break if this function wasn't there? I might be missing something

Reply:
I noticed that without this, the call to `load_from_state_dict` was calling the one defined in module.py. Once we script the model we lose access to the overridden `load_from_state_dict` so we need to call a different fn here to get around it.